utils/spacewalk-manage-channel-lifecycle

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python2
+#!/usr/bin/python
```

Comment:
This will break things. This was intentionally changed since no one has ported it to Python 3.

Reply:
True, only changes I wanted to make are:
[https://github.com/spacewalkproject/spacewalk/pull/678/files](https://github.com/spacewalkproject/spacewalk/pull/678/files)